.circleci/config.yml

```diff
@@ -113,7 +113,7 @@ commands:
           command: |
             ROOT="$PWD"
             cd tests/pytests/logs
-            tar -czf $ROOT/tests-pytests-logs.tgz *.log
+            tar -czf $ROOT/tests-pytests-logs.tgz *
```

Comment:
What else do we expect to find there?

Reply:
Sanitizer logs, which ends with `*.log.[0-9]+`